server/sonar-web/src/main/js/apps/projects/filters/Filter.js

```diff
@@ -68,7 +68,7 @@ export default class Filter extends React.PureComponent {
     );
   }
 
-  blurOnClick = (evt: Event & { currentTarget: HTMLElement }) => evt.currentTarget.blur();
+  blurOnClick = (evt /*: Event & { currentTarget: HTMLElement } */) => evt.currentTarget.blur();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)